### PR TITLE
soften language

### PR DESCRIPTION
**Pull request by ernestoongaro**
proposing changing recommended to example, forcing users to think about what they are doing for their specific scenarios

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The following fields are required when creating a Snowflake connection:
 
 ![Snowflake connection details](/img/docs/dbt-cloud/snowflake-conn-details.png)
 
-**Note:** A crucial part of working with dbt atop Snowflake is ensuring that users (in development environments) and/or service accounts (in deployment to production environments) have the correct permissions to take actions on Snowflake! Here is documentation of the [dbt Labs recommended Snowflake permissions](snowflake-permissions).
+**Note:** A crucial part of working with dbt atop Snowflake is ensuring that users (in development environments) and/or service accounts (in deployment to production environments) have the correct permissions to take actions on Snowflake! Here is documentation of some [example permissions to configure snowflake access](snowflake-permissions).
```
Comment (dbeatty10) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: Suggesting to capitalize "Snowflake":
```suggestion
**Note:** A crucial part of working with dbt atop Snowflake is ensuring that users (in development environments) and/or service accounts (in deployment to production environments) have the correct permissions to take actions on Snowflake! Here is documentation of some [example permissions to configure Snowflake access](snowflake-permissions).
```

File: website/docs/reference/snowflake-permissions.md [hunk 2]
```diff
@@ -20,4 +20,5 @@ grant select on future views in database database_name to role role_name;
 grant usage on all schemas in database database_name to role role_name;
 grant select on all tables in database database_name to role role_name;
 grant select on all views in database database_name to role role_name;
-```
\ No newline at end of file
+```
+Disclaimer
```
Comment (dbeatty10) [inline website/docs/reference/snowflake-permissions.md]: Was your intent to include more text after "Disclaimer"?

Here's the way it is rendering for me:

image

Comment (ernestoongaro) [reply website/docs/reference/snowflake-permissions.md]: Ha! Wow, I need to close my laptop. Removed in [3555772](https://github.com/dbt-labs/docs.getdbt.com/pull/2002/commits/3555772830af117c8757a4989283074f914c10fa)

**Review by matt-winkler — APPROVED** [review]
This is good - tone it down a bit

**Comment by matt-winkler** [comment]
@ernestoongaro @dbeatty10 thanks both for looking at this. I'm holding off merging until a docs team member gives the final thumbs up!

**Review by dataders — APPROVED** [review]
love it!

**Pull request merged**